plugins/woocommerce-admin/client/customize-store/assembler-hub/block-editor.tsx

```diff
@@ -92,59 +91,6 @@ export const BlockEditor = ( {} ) => {
 		[ history, urlParams, pages ]
 	);
 
-	if ( urlParams.path === '/customize-store/assembler-hub/homepage' ) {
```

Comment:
I thought we needed this to implement the homepage, but actually, in MVP, we only need to replace the home template and do not need to remove sections so I just removed these unnecessary code.